api/proto/teleport/legacy/types/types.proto

```diff
@@ -5626,6 +5626,20 @@ message PluginOktaSettings {
 
   // SSOConnectorID
   string sso_connector_id = 3;
+
+  // Import settings controls the access list import settings for Okta.
+  PluginOktaImportSettings import = 4;
+}
+
+// Defines settings for importing access lists from Okta.
+message PluginOktaImportSettings {
+  option (gogoproto.equal) = true;
+
+  // Enabled should be set to true if the Okta plugin should import access lists from Okta.
+  bool enabled = 1;
```

Comment:
Bit of bikeshedding but IMO the settings structure here has become a bit messy, there's "enable_user_sync" that controls sync of users and generically-named "import" settings that actually controls access lists (which is also actually "sync" rather than "import").

I think that having a separate "sync" section is fine, but can we at least have it contain all sync options like

```
message PluginOktaSyncSettings {
  bool sync_users = 1;
  bool sync_access_lists = 2;
  repeated string default_owners = 3;
}
```

And deprecate the standalone `bool enable_user_sync = 2`.

WDYT? A bit of scope creep for this particular change but will result in a cleaner structure I think.

Reply:
Sounds good to me.

Reply:
I've done this refactor. Let me know what you think.